src/components/booking/time.ts

```diff
@@ -0,0 +1,20 @@
+// time.ts
+import { Dayjs } from 'dayjs';
+
+export interface ReservationData {
+  startDate: string;
+  endDate: string;
+  accommodationId: string;
+}
+
+export const createReservationData = (
+  startDate: Dayjs | null,
+  endDate: Dayjs | null,
+  accommodationId: string
+): ReservationData => {
+  return {
+    startDate: startDate?.format('YYYY-MM-DD') || '',
+    endDate: endDate?.format('YYYY-MM-DD') || '',
+    accommodationId: accommodationId || '',
```

Comment:
Why do we need fallback values here? Does BE support them?

Reply:
yes BE supports